features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/WSDL/Steps/ProvideWSDL.jsx

```diff
@@ -180,13 +125,70 @@ export default function ProvideWSDL(props) {
                         </FormHelperText>
                     </FormControl>
                 </Grid>
+                <Grid item md={12}>
+                    <FormControl component='fieldset'>
+                        <FormLabel component='legend'>
+                            <React.Fragment>
+                                <sup className={classes.mandatoryStar}>*</sup>{' '}
+                                <FormattedMessage
+                                    id='Apis.Create.WSDL.Steps.ProvideWSDL.Input.type'
+                                    defaultMessage='Input type'
+                                />
+                            </React.Fragment>
+                        </FormLabel>
+                        <RadioGroup
+                            aria-label='Input type'
+                            value={apiInputs.inputType}
+                            onChange={event => inputsDispatcher({ action: 'inputType', value: event.target.value })}
+                        >
+                            <FormControlLabel value='url' control={<Radio />} label='WSDL URL' />
```

Comment:
I think we need internationalization for FormControlLabel

Reply:
Fixed